src/arg.rs

```diff
@@ -0,0 +1,64 @@
+use clap::{Arg, Command};
+
+const APP: &str = "Directory cleaner";
+pub struct Args {
+    pub types: Vec<String>,
+    pub min_size: u64,
+    pub dir: String,
+}
+
+
+pub fn parse_args() -> Args {
+    let arg = Command::new(APP)
+        .version("1.0")
+        .arg(
+            Arg::new("directory")
+                .short('d')
+                .long("dir")
+                .required(true)
+                .help("Directory to clean up")
+        )
+        .arg(
+            Arg::new("types")
+                .short('t')
+                .long("types")
+                .required(false)
+                .value_delimiter(',')
```

Comment:
kinda curious here, how would the command line args look like?

<CMD> log,bak,others...? should we make them space separated instead?

Reply:
Resolved in latest